tests/ManualModeTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Support\Facades\Event;
+use Stancl\Tenancy\Events\TenancyInitialized;
+use Stancl\Tenancy\Listeners\CreateTenantConnection;
+use \Stancl\Tenancy\Tests\Etc\Tenant;
+
+test('manual tenancy initialization works', function () {
+    Event::listen(TenancyInitialized::class, CreateTenantConnection::class);
+
+    $tenant = Tenant::create();
+
+    tenancy()->initialize($tenant);
+
+    expect(tenancy()->initialized)->toBeTrue();
```

Comment:
Does this assert anything about `CreateTenantConnection`?

Reply:
Seems like it just checks that no exception is thrown. But we want to check **that it also works**.

Reply:
> But I was confused about how to test this

It's extremely simple. The listener does only one thing: it creates a connection. So you assert that the connection doesn't exist _before_ initializing tenancy, and assert that it does exist _after_ initializing tenancy.

When you're writing a regression test, it's not enough that an exception disappears. The test you have here, as it works now, doesn't assert **anything about the behavior that should be tested according to the test name**. It just checks that an exception isn't thrown.

Reply:
> So you assert that the connection doesn't exist before initializing tenancy, and assert that it does exist after initializing tenancy.

Test updated.